test/prompt/invocation_layer/test_hugging_face.py

```diff
@@ -368,7 +372,9 @@ def test_supports(tmp_path):
         assert HFLocalInvocationLayer.supports("google/flan-t5-base")
         assert HFLocalInvocationLayer.supports("mosaicml/mpt-7b")
         assert HFLocalInvocationLayer.supports("CarperAI/stable-vicuna-13b-delta")
-        assert mock_get_task.call_count == 3
+        mock_get_task.side_effect = raise_runtime_error
```

Comment:
```suggestion
        mock_get_task.side_effect = RuntimeError
```

By doing so, you can avoid the definition of `raise_runtime_error`.

Reply:
Ok, that seems more elegant.